modules/kde/hm.nix

```diff
@@ -123,6 +125,7 @@ let
       ServiceTypes = [ "Plasma/LookAndFeel" ];
       Website = "https://github.com/danth/stylix";
     };
+    KPackageStructure = "Plasma/LookAndFeel";
```

Comment:
This line is important IMO, as Plasma 6 doesn't recognize the Global Theme if this field is missing. I was about to send a PR for that, but you beat me to it ^^

Reply:
Yeah, cause of #350. I got annoyed with home-manager crashing on startup so I went ahead and fixed what I could. No idea though how to fix the `applications.menu not in QList` error, or failiures in first start of service. Any ideas?

Reply:
No, sadly I don't know how to fix those. In my dotfiles I just put a workaround that re-tries home-manager activation 5 seconds after Plasma starts. It's hacky but the CLI tools from KDE don't seem to work without Plasma running.

My workaround works because the plasma-manager stuff (panel layouts) successfully apply at the first try and activation only fails at the very end with stylix wallpaper and theme.